### ISSUE TITLE
Feature Request - Expand the GUI to display the description of all items without scrolling

### ISSUE THREAD

**Issue by Felipefpl** [r-e-d/LeUI-SoD#3]
A common problem that affects all Infinity Engine's games is the needless scrolling we have to do thanks to the waste of space in the description of the items as we can see in the pic below:

![Sem título](https://github.com/r-e-d/LeUI-SoD/assets/5569804/4f756181-dba4-40a9-bfcd-aa999ab448b7)

Just like you did bringing back the nice/better GUI that the journal had in the classic games (that Beamdog's had to destroy without ANY good reason) it'd be a nice thing to expand the GUI to occupy the entire space between the 2 sidebars of the game vertically/horizontally to display the items' descriptions in a way where we dont need to scroll to see their descriptions. This or then change the way how the descriptions are displayed. Example: the pic above, instead of being displayed like this:

"Equiped abilities:
- Armor Class: +2, an extra +5 vs. missiles attacks.

Requires:
13 Strength

Weight: 12

Not usable by:
Bard
Druid
Mage
Monk"

It would be displayed like this:

"Equiped abilities: Armor Class +2, an extra +5 vs. missiles attacks.

Requires: 13 Strength   Weight: 12

Not usable by: Bard Druid Mage Monk and so on"

Of course in case this is possible the drawing of the item displayed in the left side could be moved to the left side or even made smaller or both to make some extra space to display the items' description horizontally. ;)

**Comment by r-e-d** [comment]
I try to keep the same window size as the default UI in this mod so I'm sorry but I won't fix this.
